tests/cli/bootsys/test_bos.py

```diff
@@ -25,24 +25,22 @@
 Unit tests for the sat.cli.bootsys.bos module.
 """
 
-from argparse import Namespace
 import logging
 import math
 import shlex
 import subprocess
-from textwrap import indent
 import unittest
+from argparse import Namespace
+from textwrap import indent
+from typing import Any
 from unittest.mock import MagicMock, Mock, call, patch
 
-from sat.cli.bootsys.bos import (
-    BOSFailure,
-    BOSLimitString,
-    BOSSessionThread,
-    BOSV2SessionWaiter,
-    boa_job_successful,
-    do_bos_shutdowns,
-    get_session_templates
-)
+from sat.apiclient import APIError
+from sat.cli.bootsys.bos import (BOSFailure, BOSLimitString, BOSSessionThread,
```

Comment:
Out of curiosity, did a code formatting tool do this for you?

Reply:
I believe it was `isort` formatting